### PR TITLE
Improve labor output for client

### PR DESCRIPTION
**Pull request by diggyk**
Added support for expanding the quest attribute to the API and added support for
expanding eventtypes to labors query.

### CODE REVIEW

File: bin/hermes [hunk 1]
```diff
@@ -98,6 +99,54 @@ def request_post(path, json):
...
+    """Print the details of a labor to standard out
+
+    Args:
+        labor: the labor to print
+        fates: all the fates so we can give suggestions
+    """
+    creation_time = parser.parse(labor["creationTime"])
+    creation_time = creation_time.replace(tzinfo=tz.tzutc())
+    creation_time = creation_time.astimezone(tz.tzlocal())
+    if labor.get("targetTime", None):
```
Comment (rra) [inline bin/hermes:112]: For future reference, `if "targetTime" in labor` is more idiomatic Python.

Comment (gmjosack) [inline bin/hermes:112]: He's checking that the value is truthy, not just that it exists. At least that was the case in the code where I suggested this.

Comment (rra) [inline bin/hermes:112]: Oh!  targetTime may be set but be set to `None`?  Then yes, this is better.

**Comment by diggyk** [comment]
Yeah, it could be set to none so I have to check.

**Pull request merged**